java/format/src/main/java/org/apache/arrow/flatbuf/BinaryView.java

```diff
@@ -1,21 +1,19 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-// 
-// http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// 
-// automatically generated by the FlatBuffers compiler, do not modify
-
```

Comment:
This shouldn't be formatted at all, right?

Reply:
The license header is added manually to the generated files. I thought it should be consistent with the other java files as well, which is why I changed those and updated the `README.md` section regarding the `format` module but the java formatting has been disabled.